docs/core/tools/dotnet-test.md

```diff
@@ -234,6 +251,12 @@ Where `Microsoft.NET.Test.Sdk` is the test host, `xunit` is the test framework.
   dotnet test --blame
   ```
 
+- Run the tests in the `test1` project providing the `-bl` argument down to `msbuild`:
+
+  ```dotnetcli
+  dotnet test ~/projects/test1/test1.csproj -bl
+  ```
```

Comment:
Also consider adding an example of passing an msbuild property

```suggestion
- Run the tests in the `test1` project providing the `-bl` (binary log) argument down to `msbuild`:

  ```dotnetcli
  dotnet test ~/projects/test1/test1.csproj -bl
  ```
```

Reply:
Isn't this a msbuild property?

Reply:
It's an MSBuild option but not a property.